src/libstd/io/mod.rs

```diff
@@ -416,6 +734,21 @@ pub trait Write {
     /// implementation do not precisely track where errors happen. For example
     /// an error on the second call to `write` will not report that the first
     /// call to `write` succeeded.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(io)]
+    /// use std::io::Write;
+    ///
+    /// let mut buffer1 = Vec::new();
+    /// let mut buffer2 = Vec::new();
+    ///
+    /// // write the output to buffer1 as we read
+    /// let mut handle = buffer1.broadcast(&mut buffer2);
+    ///
+    /// handle.write(b"some bytes").unwrap();
```

Comment:
ahh, now that this is a file, the assertion doesn't really work, unless i re-read in the file